cmd/fleetctl/get.go

```diff
@@ -61,7 +69,23 @@ func getQueriesCommand() cli.Command {
 
 				return nil
 			} else {
-				fmt.Println("[+] Getting information on a specific query is not currently supported")
+				query, err := fleet.GetQuery(name)
+				if err != nil {
+					return err
+				}
+
+				spec := specGeneric{
+					Kind:    "query",
+					Version: kolide.ApiVersion,
+					Spec:    query,
+				}
+
+				b, err := yaml.Marshal(spec)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf(string(b))
```

Comment:
Does it already have a newline?

Reply:
Also, let's avoid using `Printf` without a format string.

Reply:
It does already have a newline. I'll use `fmt.Print`.